common/config/rush/pnpm-lock.yaml

```diff
@@ -549,6 +550,7 @@ dependencies:
   '@rush-temp/cosmos': file:projects/cosmos.tgz
   '@rush-temp/data-tables': file:projects/data-tables.tgz
   '@rush-temp/dev-tool': file:projects/dev-tool.tgz
+  '@rush-temp/developer-devcenter': file:projects/developer-devcenter.tgz
```

Comment:
Please run below command to update the dependency lock file after upgrading node version:

```
> rush update && rush build -t @azure-rest/developer-devcenter
```